src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/AutoImport.props

```diff
@@ -45,6 +45,8 @@ https://github.com/dotnet/designs/blob/4703666296f5e59964961464c25807c727282cae/
     <AndroidNativeLibrary Include="**\*.so"  Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
     <JavaSourceJar        Include="**\*-source.jar;**\*-sources.jar;**\*-src.jar"   Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
     <AndroidJavaSource    Include="**\*.java" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
+    <ProguardConfiguration Include="**\*.cfg" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
```

Comment:
I wonder/fear if this is too "loose"; are these files frequently auto-generated and there are no reliable basename patterns?  What happens if "some other" `.cfg` gets globbed?  [fileformat.info](https://fileinfo.com/extension/cfg) lists ~8 different `.cfg` file types, and ProGuard *isn't* listed.

Reply:
Pity we don't recommend a particular file in the docs https://docs.microsoft.com/en-us/xamarin/android/deploy-test/release-prep/proguard?tabs=windows#customizing-proguard. We just recommend ` a .cfg file`.